inference-engine/ie_bridges/java/org/intel/openvino/Parameter.java

```diff
@@ -0,0 +1,27 @@
+package org.intel.openvino;
+
+public class Parameter extends IEWrapper {
+    public Parameter(long addr) {
+        super(addr);
+    }
+
+    public <T> T as(Class<T> cls) {
```

Comment:
Can you please share how it might be used?

Reply:
Sorry, found it.

Maybe it's better to introduce explicit methods like
```java
void get(Integer val);

void get(String val);
```

Reply:
We can't change string by reference: ```Strings are constant; their values cannot be changed after they are created.```
https://docs.oracle.com/javase/7/docs/api/java/lang/String.html

Reply:
Oh, I see. But maybe as least `asInt` and `asString`. Current class based method it too complicated